setup.py

```diff
@@ -36,6 +44,7 @@
       maintainer='LabJack Corporation',
       maintainer_email='support@labjack.com',
       classifiers=CLASSIFIERS,
+      install_requires=["future"],
```

Comment:
I think setuptools is a reasonable dependency but I don't think we should require another package to get one function used by one example.